lang/Polish.ini

```diff
@@ -124,6 +124,7 @@ FORCE_4BY3 = "Wymus 4:3"
 PRELOAD_TEXTURES = "Wstepnie Zaladuj Tekstury"
 VSYNC = "Synchronizacja Pionowa"
 UNCAPPED_FRAMERATE = "Nieograniczona Liczba Klatek"
+REDUCE_FRAMERATE_FOCUS_LOSS = "Zmniejsz liczbę klatek na sekundę, gdy okno jest nieostre"
```

Comment:
Please don't just put a text through a translator and call it a day.

I know this is mainly a code-change Pull Request, but it ends up having poor quality and bringing bad gaming experience with unclear messages.

Here *"Window Unfocused"* got translated as *„nieostre”* with the meaning closer to *"unsharp"*, *"blurry"*.

Correct translation would be either:
- *„gdy okno traci fokus”* (*"when the window loses focus"*),
- or *„gdy okno nie ma fokusa”* (*"when the window is unfocused"*),
- or *„gdy okno jest nieaktywne”* ("when the window is inactive").

Reply:
> Please don't just put a text through a translator and call it a day.
> 
> I know this is mainly a code-change Pull Request, but it ends up having poor quality and bringing bad gaming experience with unclear messages.
> 

Yeah, sorry about that. I'll look for native speakers of the non-English languages to ensure the text is replaced with ones that actually make sense (German has been done like that so far, currently getting Spanish done).

> Correct translation would be either:
> 
>     * _„gdy okno traci fokus”_ (_"when the window loses focus"_),
> 
>     * or _„gdy okno nie ma fokusa”_ (_"when the window is unfocused"_),
> 
>     * or _„gdy okno jest nieaktywne”_ ("when the window is inactive").

First one seems seems good. Thanks!